packages/typo3-docs-theme/resources/template/structure/header/additionalMetaData.html.twig

```diff
@@ -0,0 +1,9 @@
+{%- set officialManualShortcode = getSettings('official_manual_interlink_shortcode') -%}
+{%- set isOfficialManualPredefined = getSettings('official_manual_interlink_predefined') -%}
+
+{%- if isOfficialManualPredefined -%}
+    <meta name="t3docs-guides:interlink-predefined-manual" content="true">
```

Comment:
What are these both meta tags about? Where are they used?

Reply:
Not needed anymore, removed